examples/chrono/chrono_main.c

```diff
@@ -393,7 +393,7 @@ int main(int argc, FAR char *argv[])
         {
           /* Copy the initial value */
 
-          strncpy(str, "00:00.0", 7);
+          strlcpy(str, "00:00.0", sizeof(str));
```

Comment:
so this code also generated a warning? very strange, because sizes seems to be well calculated.

Reply:
Yes.